lib/process-services/src/lib/form/widgets/content-widget/attach-file-widget-dialog.component.spec.ts

```diff
@@ -162,6 +163,18 @@ describe('AttachFileWidgetDialogComponent', () => {
             });
         });
 
+        it('[C594015] should not be able to choose a folder', () => {
+            spyOn(widget,'onSelect');
+            const fakeFolderNode: Node = new Node({ id: 'fakeFolder', isFile: false, isFolder: true});
+
+            contentNodePanel.componentInstance.onCurrentSelection([ { entry: fakeFolderNode }]);
+            fixture.detectChanges();
+
+            const chooseButton: HTMLButtonElement = element.querySelector('button[data-automation-id="attach-file-dialog-actions-choose"]');
```

Comment:
the automation id should be unique, so you don't need specifying the "button" there

Reply:
thx